modular_skyrat/modules/modular_reagents/code/reagents/medicine.dm

```diff
@@ -25,6 +25,84 @@
 	M.adjustOrganLoss(ORGAN_SLOT_HEART,3 * REM * delta_time, 80)
 	..()
 
+
+/datum/reagent/medicine/ataraxydone
+	name = "Ataraxydone"
+	description = "A stasis-inducing chemical used for exotic surgeries or emergency medicine."
+	taste_description = "liquid"
+	reagent_state = LIQUID
+	color = "#e0d394" // 224, 211, 148
+	metabolization_rate = 0.2 * REAGENTS_METABOLISM
+	overdose_threshold = 20
+	ph = 7
+	purity = REAGENT_STANDARD_PURITY
+	creation_purity = REAGENT_STANDARD_PURITY
+	impure_chem = /datum/reagent/impurity/ataraxydone
+	chemical_flags = REAGENT_CAN_BE_SYNTHESIZED | REAGENT_DEAD_PROCESS | REAGENT_IGNORE_STASIS
+
+/datum/reagent/medicine/ataraxydone/on_mob_dead(mob/living/carbon/owner, delta_time)
+	.=..()
+	owner.apply_status_effect(STATUS_EFFECT_STASIS_MOBILE, STASIS_CHEMICAL_EFFECT)
+	metabolization_rate = 0.1 * REAGENTS_METABOLISM
+
+/datum/reagent/medicine/ataraxydone/on_mob_life(mob/living/carbon/owner, delta_time, times_fired)
+	var/ataraxydone_speed = max(1, min(current_cycle/10, 8))
+	if(owner.resting || owner.stat >= SOFT_CRIT)
+		owner.apply_status_effect(STATUS_EFFECT_STASIS_MOBILE, STASIS_CHEMICAL_EFFECT)
+		owner.add_movespeed_modifier(/datum/movespeed_modifier/reagent/ataraxyprone)
+		current_cycle += 1
+	else
+		owner.remove_status_effect(STATUS_EFFECT_STASIS_MOBILE, STASIS_CHEMICAL_EFFECT)
+		owner.remove_movespeed_modifier(/datum/movespeed_modifier/reagent/ataraxyprone)
+	if(current_cycle >= 10)
+		owner.add_or_update_variable_movespeed_modifier(/datum/movespeed_modifier/reagent/ataraxydone, multiplicative_slowdown = ataraxydone_speed)
+	if(owner.stat < SOFT_CRIT)
+		switch(current_cycle)
+			if(1)
+				to_chat(owner, span_notice("You feel a comfortable stillness begin to set in throughout your body."))
+			if(12)
+				to_chat(owner, span_notice("It'd be easy to lay down for a bit..."))
+			if(32)
+				to_chat(owner, span_notice("It's getting hard to move..."))
+			if(52)
+				to_chat(owner, span_notice("...have I even been breathing..?"))
+			if(48 to 79)
+				owner.adjust_drowsyness(4 * REM * delta_time)
+			if(80 to INFINITY)
+				owner.Sleeping(50 * REM * delta_time)
+				. = TRUE
+	..()
+
+/datum/reagent/medicine/ataraxydone/on_mob_delete(mob/living/carbon/owner, amount)
+	.=..()
+	owner.remove_status_effect(STATUS_EFFECT_STASIS_MOBILE, STASIS_CHEMICAL_EFFECT)
+	owner.remove_movespeed_modifier(/datum/movespeed_modifier/reagent/ataraxydone)
+	owner.remove_movespeed_modifier(/datum/movespeed_modifier/reagent/ataraxyprone)
+	REMOVE_TRAIT(owner, TRAIT_PACIFISM, "ataraxydone")
+
+/datum/reagent/medicine/ataraxydone/overdose_process(mob/living/owner, delta_time, times_fired)
+	if(owner.stat != DEAD)
+		if(current_cycle >= 16 && !HAS_TRAIT(owner, TRAIT_PACIFISM))
+			if(DT_PROB(15, delta_time))
+				to_chat(owner, span_notice("...what's the point of fighting, anyway?"))
+				ADD_TRAIT(owner, TRAIT_PACIFISM, "ataraxydone")
```

Comment:
"ataraxydone" here should be a define, like `TRAIT_SOURCE_ATARAXYDONE_OD` (or something like that, I forget what the standard way of writing those source defines is, I recommend checking that yourself).